inference/generativeai/llm-workshop/lab10-falcon-40b-and-7b/falcon-7b-deepspeed.ipynb

```diff
@@ -0,0 +1,1165 @@
+{
```

Comment:
<p>Clear output to keep clean</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>